indico/modules/rb/models/locations.py

```diff
@@ -162,18 +161,5 @@ def get_equipment_by_name(self, name):
         return self.equipment_types.filter_by(name=name).first()
 
     def get_buildings(self):
-        building_rooms = defaultdict(list)
-        for room in self.rooms:
-            building_rooms[room.building].append(room)
-
-        buildings = []
-        for building_name, rooms in building_rooms.iteritems():
-            room_with_lat_lon = next((r for r in rooms if r.longitude and r.latitude), None)
-            if not room_with_lat_lon:
-                continue
-            buildings.append({'number': building_name,
-                              'title': _(u'Building {}'.format(building_name)),
-                              'longitude': room_with_lat_lon.longitude,
-                              'latitude': room_with_lat_lon.latitude,
-                              'rooms': [r.to_serializable('__public_exhaustive__') for r in rooms]})
-        return buildings
+        from indico.modules.rb_new.util import get_buildings
```

Comment:
let's keep this one and the tests as they were before - we can just remove them later when removing the old RB stuff